_docs/technical/2017-01-04-updater.md

```diff
@@ -93,14 +99,17 @@ As nodes deserialize
 they modify mempool as implemented
 [here](https://github.com/input-output-hk/cardano-sl/blob/22360aa45e5dd82d0c87872d8530217fc3d08f4a/src/Pos/Update/MemState/Functions.hs#L40).
```

Comment:
This link refers to an old function which was removed few weeks ago. Maybe change it to a newer version? I am not sure if it's possible now (without updating links in 100500 places).